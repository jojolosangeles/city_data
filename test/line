set -x
python ../code/main.py scripts/line.txt > actual/line.out
diff expected/line.out actual/line.out
diff -b expected/all_areas.Year.Age.stacked_line.png actual
