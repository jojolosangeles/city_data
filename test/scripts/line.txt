all_areas = expected/all_areas.csv
all_areas.Age <= [("<25" if (x < 25) else "<35" if (x < 35) else "<45" if (x < 45) else "<55" if (x < 55) else "<65" if (x < 65) else "<75" if (x < 75) else "Old") for x in all_areas['Victim Age']]
all_areas.drop Unnamed: 0, Date Occurred, Time Occurred, Area Name, Date, Weekday, Hour
all_areas.drop Victim Age, Victim Sex, Victim Descent, Address, Cross Street
all_areas.columns
all_areas.xrow Year=2019
all_areas.Year.Age => stacked

