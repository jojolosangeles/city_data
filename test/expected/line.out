Test: scripts/line.txt
> all_areas = expected/all_areas.csv
Loaded expected/all_areas.csv into DataFrame 'all_areas', 453817 rows
> all_areas.Age <= [("<25" if (x < 25) else "<35" if (x < 35) else "<45" if (x < 45) else "<55" if (x < 55) else "<65" if (x < 65) else "<75" if (x < 75) else "Old") for x in all_areas['Victim Age']]
> all_areas.drop Unnamed: 0, Date Occurred, Time Occurred, Area Name, Date, Weekday, Hour
Dropped ['Unnamed: 0', 'Date Occurred', 'Time Occurred', 'Area Name', 'Date', 'Weekday', 'Hour'] from DataFrame 'all_areas'
> all_areas.drop Victim Age, Victim Sex, Victim Descent, Address, Cross Street
Dropped ['Victim Age', 'Victim Sex', 'Victim Descent', 'Address', 'Cross Street'] from DataFrame 'all_areas'
> all_areas.columns
['Year' 'Age']
> all_areas.xrow Year=2019
451531 rows remaining
> all_areas.Year.Age => stacked
Saved PNG stacked lines: ./actual/all_areas.Year.Age.stacked_line.png
> 
